examples/qwik/README.md

```diff
@@ -0,0 +1,66 @@
+# Qwik App ⚡️
```

Comment:
Can we trim this example down and remove:

- README
- Static assets
- CSS files and use utilities instead

to keep it minimal as other examples. Thanks.

Reply:
Yes, of course. :)